releasenotes/notes/prepare-0.24-423de82722d2e31a.yaml

```
@@ -0,0 +1,67 @@
+---
+prelude: |
+  This release is a major feature release that includes numerous new features
+  and bugfixes.
+
+  This release is the final release with support for running Qiskit
+  with Python 3.7. Starting in the next minor version release Python >=3.8 will
+  be required to run Qiskit.
+
+  The highlights of this release:
+
+  QuantumInstance, OpFlow, and algorithms usage deprecation
+  ---------------------------------------------------------
+
+  This release officially deprecates the :class:`~QuantumInstance` class (and
```

I'm never sure when the objects are automatically discovered, so maybe this is not needed
```suggestion
  This release officially deprecates the :class:`~.QuantumInstance` class (and
```